ddtrace/contrib/internal/aiohttp/middlewares.py

```diff
@@ -132,19 +132,30 @@ def finish_request_span(request, response):
         response_headers=response.headers,
         route=route,
     )
+    if type(response) is web.StreamResponse and not response.task.done():
+        request_span_var.set(request_span)
+        request.task.add_done_callback(span_done_callback)
```

Comment:
TODO: 
Investigate why the spans take longer to close: https://gitlab.ddbuild.io/DataDog/apm-reliability/dd-trace-py/-/jobs/726153315 
> assert 0.499582929 < 0.05

(Since this means either the test expectation needs to be adjusted, or there's an unexpected behavior elsewhere)